pkg/apiserver/api_installer.go

```diff
@@ -379,18 +384,22 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 				Filter(m).
 				Doc("partially update the specified "+kind).
 				Consumes(string(api.JSONPatchType), string(api.MergePatchType), string(api.StrategicMergePatchType)).
-				Operation("patch" + kind).
+				Operation("patch"+kind).
 				Produces(append(storageMeta.ProducesMIMETypes(action.Verb), "application/json")...).
-				Reads(versionedObject)
+				Returns(http.StatusOK, "OK", versionedObject).
+				Reads(versionedObject).
```

Comment:
This depends on the content type. For JSONPatchType, it should be something like Patch:
https://github.com/evanphx/json-patch/blob/master/patch.go#L27

Would it be better to make this more generic, at least for now?